libkernelflinger/security.c

```diff
@@ -216,6 +218,123 @@ struct rot_data_t* get_rot_data()
 	return &rot_data;
 }
 
+char* strrpl(CHAR8 *in, const char *src, const char *dst){
+    CHAR8 *p = in;
+    int n = 0;
+    while(*p !='\0'){
+        if(strncmp(p, src, strlen(src)) != 0){
+            in[n] = *p;
+        }else{
+            in[n] = *dst;
+        }
+        p++;
+        n++;
+    }
+    return in;
+}
+
+CHAR8* set_attestation_ids(CHAR8* input){
+    const char *delim2 = "=";
+    CHAR8 *savedPtr2;
+    const char *d1 = ",";
+    const char *d2 = " ";
+    CHAR8 *output = strtok_r(token, delim2, (char **)&savedPtr2);
+    output = strtok_r(NULL, delim2, (char **)&savedPtr2);//Get the string after the equal sign
```

Comment:
I think you don't need to call strtok_r again, after the first strtok_r, savedPtr2 already point to the id value.